### PR TITLE
Add new profile for pavucontrol

### PR DESCRIPTION
**Pull request by glitsj16**

### CODE REVIEW

File: etc/pavucontrol.profile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+include disable-programs.inc
+include disable-xdg.inc
+
+include whitelist-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+machine-id
```
Comment (SkewedZeppelin) [inline etc/pavucontrol.profile]: remove machine-id

File: etc/pavucontrol.profile [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+novideo
+protocol unix
+seccomp
+shell none
+
+disable-mnt
+private-bin pavucontrol
+private-cache
+private-dev
+private-etc alternatives,asound.conf,fonts,pulse
```
Comment (SkewedZeppelin) [inline etc/pavucontrol.profile]: add machine-id

File: etc/pavucontrol.profile (hunk 1 above)
Comment (glitsj16) [reply etc/pavucontrol.profile]: Done via https://github.com/netblue30/firejail/pull/2511/.

File: etc/pavucontrol.profile (hunk 2 above)
Comment (glitsj16) [reply etc/pavucontrol.profile]: Done via https://github.com/netblue30/firejail/pull/2511/.

**Pull request merged**